lib/util/git/index.ts

```diff
@@ -199,6 +200,75 @@ export async function validateGitVersion(): Promise<boolean> {
   return true;
 }
 
+export async function enableCredentialStore(): Promise<void> {
+  const currentHelper = await simpleGit().getConfig(
+    'credential.helper',
+    'global',
+  );
+
+  if (!currentHelper.values.includes('store')) {
+    await simpleGit().addConfig('credential.helper', 'store', true, 'global');
+  }
+}
+
+export async function updateCredentialStore(url: string): Promise<void> {
+  const credentialsPath = upath.join(homedir(), '.git-credentials');
```

Comment:
@rarkins this won't work when home is read only.

Reply:
The latter write attempt fails in the case of a read-only home, not due to the determination of `credentialsPath`. The `readFile()` is already wrapped in a try/catch block, so adding another one for `writeFile()` should be sufficient. Do you agree?

Anyway, I will also add a note in the docs about write permissions for `~/.git-credentials`.